client/state/jetpack/settings/utils.js

```diff
@@ -48,6 +48,9 @@ export const normalizeSettings = ( settings ) => {
 export const sanitizeSettings = ( settings ) => {
 	return Object.keys( settings ).reduce( ( memo, key ) => {
 		switch ( key ) {
+			// Jetpack's settings endpoint in version 4.9 does not support receiving 'akismet' among the settings
+			case 'akismet':
```

Comment:
Maybe we can combine all settings that we ignore together above a single `break`?